vpdq/version.txt

```diff
@@ -0,0 +1 @@
+0.0.20
```

Comment:
@schatten - Do you think it's worth starting with this pattern? I don't know exactly how the one for pytx works, would love your double check here

Reply:
Let's start with 0.0.1, unless we already have 19 prior versions. 

The major.minor.patch or major.minor.micro versioning is [compatible](https://peps.python.org/pep-0440/#final-releases) with pypi's instructions.

Common package managers will also be able to do version pinning and open ended declarations like >=1.2.3, so this works!